src/inmanta/env.py

```diff
@@ -1054,8 +1054,11 @@ async def async_run_command_and_stream_output(
         of the sub-process.
         """
         full_output = []
+        # We use close_fds=False to use an alternative path to spawn to avoid
+        # the bug https://github.com/python/cpython/issues/103911#issuecomment-2333963137
+        # This uses spawn instead of fork+execve
         process = await asyncio.create_subprocess_exec(
-            cmd[0], *cmd[1:], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, env=env_vars
+            cmd[0], *cmd[1:], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, env=env_vars, close_fds=False
```

Comment:
I can only review this part superficially because I don't understand all intricacies of what's going on.

Reply:
same as everyone 'm afraid